specification/iothub/resource-manager/Microsoft.Devices/preview/2022-04-30-preview/iothub.json

```diff
@@ -4452,6 +4574,22 @@
           "description": "The timestamp of resource last modification (UTC)"
         }
       }
+    },
+    "RootCertificateProperties": {
+      "description": "This property store root certificate related information",
+      "type": "object",
+      "properties": {
+        "enableRootCertificateV2": {
+          "description": "This property when set to true, hub will use G2 cert; while it's set to false, hub uses Baltimore Cert.",
+          "type": "boolean"
+        },
```

Comment:
Should this better be an enum (if you later have more than G2 and Baltimore)?

Reply:
No, the feature will only be in preview version, we will retired the API soon after the deadline of Baltimore cert sunset.